CHANGELOG.md

```diff
@@ -24,7 +24,90 @@ Current
 - [`ResponseValidationException` uses `Response.StatusType` rather than `Response.Status`](https://github.com/yahoo/fili/pull/96)
     * `Response.StatusType` is the interface that `Response.Status` implements.
     * This will have no impact on current code in Fili that uses `ResponseValidationException`, and it allows customers to inject http
-        codes not included in `Response.Status`. 
+        codes not included in `Response.Status`.
+         
+- [Removed "provided" modifier for SLF4J and Logback dependencies in the Wikipedia example](https://github.com/yahoo/fili/pull/102)
+
+- [Updated dependencies](https://github.com/yahoo/fili/pull/103)
+
+    Unless otherwise noted, all dependency upgrades are for general stability and performance improvement. The called-
+    out changes are only those that are likely of interest to Fili. Any dependency upgrade for which a changelog could
+    not be found has not been linked to one, otherwise all other upgrades include a link to the relevant changelog.
+
+    * [Gmaven plugin 1.4 -> 1.5](https://github.com/groovy/gmaven/compare/gmaven-1.4...gmaven-1.5)
+    * [Guava 16.0.1 -> 20.0](https://github.com/google/guava/wiki/ReleaseHistory)
+    * [Jedis 2.7.2 -> 2.9.0](https://github.com/xetorthio/jedis/releases):
+        - Geo command support and binary mode support
+        - ZADD support
+        - Ipv6 and SSL support
+        - Other assorted feature and Redis support upgrades
+    * [Redisson 2.2.13 -> 3.1.0](https://github.com/redisson/redisson/blob/master/CHANGELOG.md):
+        - Support for binary stream in and out of Reddison
+        - Lots of features for distributed data structure capabilities
+        - Can make fire-and-forget style calls in ack-response-only modes
+        - Many fixes and improvements for PubSub features
+        - Support for command timeouts
+        - Fixed bug where connections did not always close when RedisClient shut down
+        - Breaking API changes:
+        - Moved config classes to own package
+        - Moved core classes to api package
+        - Moved to Redisson's RFuture instead of netty's Future
```

Comment:
These (starting at [line 52](https://github.com/yahoo/fili/pull/103/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR52)) should be indented one more level.